src/components/tooltip/tooltip.hbs

```diff
@@ -12,10 +12,11 @@
 </div>
 <div id="unique-tooltip" data-floating-menu-direction="bottom" class="bx--tooltip" data-avoid-focus-on-open>
   <span class="bx--tooltip__caret"></span>
-  <p>This is some tooltip text. This box shows the maximum amount of text that should appear inside. If more room is needed
+  <p>This is some tooltip text. This box shows the maximum amount of text that should appear inside. If more room is
+    needed
     please use a modal instead.</p>
-  <div>
-    <a href="#" class="bx--link">Learn More</a>
+  <div class="bx--tooltip__footer">
+    <a href=" #" class="bx--link">Learn More</a>
```

Comment:
Was the addition of space before `#` an accident?

Reply:
Yes, let me fix that 😄